posts/telemetry-yasmin.adoc

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: "Tracing your microservices made easy with MicroProfile Telemetry 1.0"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yasmin-aumeeruddy
+author_github: https://github.com/yasmin-aumeeruddy
+seo-title: Logs aren't always enough: Tracing your microservices made easy with MicroProfile Telemetry 1.0
+seo-description: Distributed tracing helps DevOps teams keep track of requests between microservices. MicroProfile adopts OpenTelemetry Tracing to allow developers to observe requests across their distributed systems.
+blog_description: Distributed tracing helps DevOps teams keep track of requests between microservices. MicroProfile adopts OpenTelemetry Tracing to allow developers to observe requests across their distributed systems.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Tracing your microservices made easy with MicroProfile Telemetry 1.0
+
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+The use of microservice architecture may increase difficultly to see how services depend on or affect other services. Consequently, making it harder to find the source of latency or inaccuracy. 
+
+One way to increase observability of an application is by emitting traces. link:https://opentelemetry.io/[OpenTelemetry] is a set of APIs, SDKs, tooling and integrations that are designed for the creation and management of telemetry data such as traces, metrics, and logs. link:https://projects.eclipse.org/projects/technology.microprofile/releases/microprofile-telemetry-1.0/plan[MicroProfile Telemetry 1.0] adopts OpenTelemetry Tracing so your MicroProfile applications can benefit from both manual and automatic traces! 
+
+* *Traces* represent requests and consist of multiple spans.
```

Comment:
for accessibility compliance, use the following definition list syntax instead of inline bold:

```suggestion
Traces:: 
Traces represent requests and consist of multiple spans.
```
 similar for the other items in this list